libs/ui/src/lib/modal/modal.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from 'clsx';
+import { useMemo } from 'react';
+
+export interface IModal {
+  open: boolean;
+  children: React.ReactNode;
+}
+
+export function Modal({ open, children }: IModal) {
+  const modalClass = useMemo(
+    () =>
+      clsx('modal bg-black/50', { 'modal-open': open, 'modal-closed': !open }),
```

Comment:
Looks like we're using the Daisy UI modal syntax here. What about just using Tailwind instead so we have more granular control? Something like this maybe?

```suggestion
      clsx('fixed top-0 left-0 w-full h-full bg-black/50', { 'modal-open': open, 'modal-closed': !open }),
```

Reply:
I was wondering which lib these `css` are from in the permission modal!😅 If we remove`modal`, the `modal-open` and `modal-close` doesn't seem to work so I put modal back for now.
Do we want to use the css in https://tailwind-elements.com/docs/standard/components/modal/?

Reply:
I was wondering about that. All good, we can start with Daisy UI and transition off. We actually don't use the tailwind component library though. Sadly we don't have access to that CSS link.